jobs.html

```diff
@@ -2,18 +2,28 @@
 layout: default
 title: Jobs with Compiler
 ---
-<h1>Jobs</h1>
+<div class="jobs container mt-5">
+    <div class="row">
+        <div class="col-md-8 offset-md-2 mt-5">
+            <span class="label rounded p-2">Careers</span>
```

Comment:
I think something like this is normally called a `pill`

Reply:
Renamed it in [6036943](https://github.com/compilerla/compiler.la/pull/87/commits/6036943f82145b085d51128f17206804e54f3e61)